### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by andreandersson** [HaylLtd/cordova-plugin-background-geolocation#133] [General]
Hello!

Is the next release planned anytime soon? I'm, surprise, surprise, interested in getting #111 into a release...

**Comment by HarelM** [comment]
Can you send a PR with version change by any chance? There are 3 places to change IIRC...

**Reply by andreandersson** [reply]
Done with #134 , hopefully I didn't miss something... :)

**Reply by HarelM** [reply]
I've just released 2.0.8, can you check that everything is working as expected?

**Reply by andreandersson** [reply]
Rolled out to our app Friday, and it seems to be working. Crash reports are down 100%, and ANR-reports are around 99.97% down after the update. This was the only component changed in our release.

Hoping the trend continues when the rollout continues.
